js/components/graph/Graph.js

```diff
@@ -33,6 +34,8 @@ export class Graph extends React.Component {
       hybridsJSON: [],
       boolsJSON: [],
       edgesJSON: [],
+      edgeMissingStatus: {},
```

Comment:
This status isn't actually used anywhere, and can be deleted, along with all of the `updateEdgeMissingStatus` code.